api/client/proto/authservice.proto

```diff
@@ -1094,6 +1094,31 @@ message CompleteAccountRecoveryRequest {
     }
 }
 
+// CreateAccountRecoveryCodesRequest is a request to create new set of recovery codes for a user,
+// replacing and invalidating any previously existing codes. Recovery codes can only be given to
+// users who meet the following requirements:
+//  - cloud feature is enabled
+//  - username is in valid email format
+message CreateAccountRecoveryCodesRequest {
+    // TokenID is the ID of a user token that will be used to verify this request.
```

Comment:
Does it have to be a specific kind of token?

Reply:
I left it open to accept `privilege token` for later.

Reply:
Should we add that to the docs?